vizro-core/src/vizro/_vizro.py

```diff
@@ -201,12 +201,11 @@ def _make_resource_spec(path: Path) -> _ResourceSpec:
     BASE_EXTERNAL_URL = f"https://cdn.jsdelivr.net/gh/mckinsey/vizro@{_git_branch}/vizro-core/src/vizro/"
 
     # Get path relative to the vizro package root, where this file resides.
-    relative_path = path.relative_to(Path(__file__).parent)
+    # This must be a posix path to work on Windows, so that we convert all \ to / and routing works correctly.
+    # See https://github.com/mckinsey/vizro/issues/836.
+    relative_path = PurePosixPath(path.relative_to(Path(__file__).parent))
```

Comment:
I like the refactoring you did in this file a few weeks ago. That made it a lot easier to make this change now 👍 👍

Reply:
Should we apply PurePosixPath to `VIZRO_ASSETS_PATH = Path(__file__).with_name("static")` from `_constants.py` too? Just by looking at the code I don't think we should do that, but it's just heads up in case you forgot.

Reply:
Thanks for checking! Indeed we shouldn't change it there because a `PurePath` doesn't allow for various operations we need with `VIZRO_ASSETS_PATH` like `read_text`. It's best to leave as concrete paths until the point of use when it needs to be converted.